From: Douglas Anderson <dianders@chromium.org>
Subject: watchdog/hardlockup: remove softlockup comment in touch_nmi_watchdog()
Date: Fri, 26 May 2023 18:41:35 -0700

In the patch ("watchdog/hardlockup: add comments to touch_nmi_watchdog()")
we adjusted some comments for touch_nmi_watchdog().  The comment about the
softlockup had a typo and were also felt to be too obvious.  Remove it.

Link: https://lkml.kernel.org/r/20230526184139.5.Ia593afc9eb12082d55ea6681dc2c5a89677f20a8@changeid
Signed-off-by: Douglas Anderson <dianders@chromium.org>
Suggested-by: Petr Mladek <pmladek@suse.com>
Reviewed-by: Petr Mladek <pmladek@suse.com>
Cc: Christophe Leroy <christophe.leroy@csgroup.eu>
Cc: "David S. Miller" <davem@davemloft.net>
Cc: Michael Ellerman <mpe@ellerman.id.au>
Cc: Nicholas Piggin <npiggin@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/nmi.h |    4 ----
 1 file changed, 4 deletions(-)

--- a/include/linux/nmi.h~watchdog-hardlockup-remove-softlockup-comment-in-touch_nmi_watchdog
+++ a/include/linux/nmi.h
@@ -140,10 +140,6 @@ static inline void touch_nmi_watchdog(vo
 	 */
 	arch_touch_nmi_watchdog();
 
-	/*
-	 * Touching the hardlock detector implicitly resets the
-	 * softlockup detector too
-	 */
 	touch_softlockup_watchdog();
 }
 
_
